scenes/menu/main_menu.gd

```diff
@@ -345,6 +362,7 @@ func _on_char_select_select_character(char_id):
 
 func _on_change_player_character_button_pressed(is_player : bool):
 	var char_id = player_selected_character
+	print("what the fuck")
```

Comment:
let me know if you have a question....

Reply:
lol i meant to delete all these, they were for debugging and that is not a reference to your code